zeus/vcs/backends/git.py

```diff
@@ -232,14 +252,21 @@ def log(
 
     def export(self, sha, update_if_exists=False) -> str:
         cmd = ["diff", "%s^..%s" % (sha, sha)]
-        self.ensure(update_if_exists=update_if_exists)
-        result = self.run(cmd)
+        with sentry.Span("vcs.export", self.remote_url) as par_span:
+            par_span.set_tag("sha", sha)
+            par_span.set_tag("backend", "git")
```

Comment:
you could imagine my use case for wanting all 'vcs.*' to have 'backend=X' on them, so you could aggregate if there were different backends and analyze many transactions